src/wled2.html

```diff
@@ -5,17 +5,28 @@
     defaults: {
       address: { value: "", required: true },
       brightness: { value: 128 },
+      enableBrightness: { value: true },
       delay: { value: 0, validate: RED.validators.number() },
+      enableDelay: { value: true },
       color1: { value: "#FFFFFF" },
+      enableColor1: { value: true },
       color2: { value: "#FFFFFF" },
+      enableColor2: { value: true },
       color3: { value: "#FFFFFF" },
+      enableColor3: { value: true },
       effect: { value: 0 },
+      enableEffect: { value: true },
       effectIntensity: { value: 128 },
+      enableEffectIntensity: { value: true },
       effectSpeed: { value: 128 },
+      enableEffectSpeed: { value: true },
       name: { value: "" },
       palette: { value: "0" },
-      preset: { value: "" },
+      enablePalette: { value: true },
+      preset: { value: null },

```

Comment:
What will "null" do on the number selector? Will it show empty but have enabled checked? Will that cause problems for someone who just drops in the node and doesn't want a preset?

Reply:
In the code we do a check for both `enablePreset` and `mergedPayload.preset`, with the enable overridden by an incoming msg payload's preset having a value that passes `if (payload.preset)`. 

So if you drop in the node and don't want a preset, it will pass on enablePreset, and fail on `mergedPayload.preset` being set to undefined from the `which` function

Reply:
But what is the result for the number control and the checkbox in the UI? I believe this winds up with the enable checkbox set but null bound to the number control. That seems... weird.

Seems like the initial experience for a new node should be:

1. Enable checkbox for preset is unchecked
2. Preset number defaults to 1

Reply:
Totally good with point 1

What about a default to -1 to show it exists to the user for point 2?